src/main/java/organice/logic/commands/MatchCommand.java

```diff
@@ -88,7 +92,15 @@ private CommandResult executeMatchNric(Nric patientNric, Model model) {
                 patient = model.getPatient(patientNric);
                 model.matchDonors(patient);
 
-                CommandResult commandResult = new CommandResult(MESSAGE_SUCCESS);
+
+                CommandResult commandResult;
```

Comment:
Change only if have time to complete: Throw error if command result is still null at the end. Right now there's no guard clause for that.

Reply:
Added

Reply:
Not changed because of lack of time